docs/oauth2.rst

```diff
@@ -115,12 +115,18 @@ For End Users: Using OAuth Client ID
 This is the case where your application or a script is accessing spreadsheets on behalf of an end user. When you use this scenario, your application or a script will ask the end user (or yourself if you're running it) to grant access to the user's data.
 
 1. :ref:`enable-api-access` if you haven't done it yet.
```

Comment:
you could update the numbers as well (different commit perhaps ?) to match the sphinx format that uses `#` for lists.

Reply:
Thanks for the review @lavigne958 .
Sorry, I'm not sure what numbers should be updated. I think the enumeration is fine, as seen in the capture provided in my previous comment.
Can you give a little more details about what should be updated?

Reply:
I found it odd the mix of numbers and `#` so I checked onlin the doc and it says the ordered list is build using `#`. Then I found today that you can start with a number then the `#` will auto increment. So It's all good at the end. :wink: